EIPS/blockhash_refactoring.md

```diff
@@ -0,0 +1,84 @@
+### Preamble
+
+    EIP: <to be assigned>
+    Title: Blockhash refactoring
+    Author: Vitalik Buterin
+    Type: Standard Track
+    Category: Core
+    Status: Draft
+    Created: 2017-02-10
+    
+### Summary
+
+Stores blockhashes in the state, reducing the protocol complexity and the need for client implementation complexity in order to process the BLOCKHASH opcode. Also extends the range of how far back blockhash checking can go, with the side effect of creating direct links between blocks with very distant block numbers, facilitating much more efficient initial light client syncing.
+
+### Parameters
+
+* `METROPOLIS_FORK_BLKNUM`: TBD
```

Comment:
Now, this should say `BYZANTIUM_FORK_BLKNUM`.

Reply:
Nope, should say `CONSTANTINOPLE_FORK_BLNUM`

Reply:
Right, it has changed.